Framework/Algorithms/src/CreatePeaksWorkspace.cpp

```diff
@@ -36,58 +37,76 @@ void CreatePeaksWorkspace::init() {
   declareProperty(std::make_unique<WorkspaceProperty<IPeaksWorkspace>>(
                       "OutputWorkspace", "", Direction::Output),
                   "An output workspace.");
+  // explicit control of output peak workspace tyep
+  // Full: standar peak workspace
+  // Lean: LeanElasticPeakWorkspace
+  const std::vector<std::string> peakworkspaceTypes{"Full", "Lean"};
+  declareProperty(
```

Comment:
We should be consistent with this property between different algorithms, for FindPeaksMD #30953 I had the property `OutputPeakType` with the options `Peak` and `LeanElasticPeak`. I will the same property with my upcoming changes for `PredictPeaks`.

Reply:
I will update the docs and option list as suggested.